core/templates/components/common-layout-directives/common-elements/classroom-navigation-links.component.html

```diff
@@ -4,6 +4,7 @@
   </ng-container>
   <ng-template #content>
     <div class="classroom-navigation-links-content">
+      <span *ngIf="classroomSummaries.length === 0">No classrooms available. To create classrooms, visit the <a href="/classroom-admin">classroom admin</a> page.</span>
```

Comment:
Should we instruct developers to get the necessary roles (curriculum admin, I assume) in order to access the classroom admin page?

Reply:
I guess we can add a text saying that you need curriculum admin role but adding steps for it will be too much since we already have those things on our wiki page.

Reply:
Done